util/reconciler/defaults.go

```diff
@@ -37,7 +37,7 @@ const (
 	// DefaultMachineControllerWaitForRunningTimeout is the default timeout if instance is not running.
 	DefaultMachineControllerWaitForRunningTimeout = 20 * time.Minute
 	// DefaultMachineControllerRetryDelay is the default requeue delay if there is an error.
-	DefaultMachineControllerRetryDelay = 10 * time.Second
+	DefaultMachineControllerRetryDelay = 30 * time.Second
```

Comment:
if this bump is specifically for 429s could we maybe create a retry case just for too many requests so other transient errors can be retried more quickly?

Reply:
The others are 500 errors, I'm not sure we want to retry every 10 seconds in those cases either but can.

Reply:
if the errors are actually transient a quick retry should generally be appropriate, if you think we need to reevaluate retries for that status then I think we can do that as well